README.md

```diff
@@ -36,6 +36,9 @@ the organization.
 - [node-auto-test](https://github.com/nodejs/node-auto-test)
   - A [nodejs/node](https://github.com/nodejs/node) clone for testing tools
     and bots.
+- [node-core-utils](https://github.com/nodejs/node-core-utils)
+  - Command line tools for Collaborators. Generating and formatting PR 
+  metadatas, and much more.
```

Comment:
I think `Command line tools for Collaborators.` is pretty enough here..also `metadatas` sounds a bit weird?